_data/finance.yml

```
@@ -156,6 +156,13 @@ websites:
       img: levelup.png
       tfa: No
 
+    - name: Linxo
+      url: https://wwws.linxo.com
```

Please change the url to:
```suggestion
      url: https://www.linxo.com/
```